.github/workflows/release-build.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will run tests using node and then publish a package when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Build UMD and Translations
```

Comment:
Just realized that this pipeline can be triggered when the branch `changeset-release/main` merged to `main`, but we can do it later.